operator/controllers/common/helperfunctions.go

```diff
@@ -103,3 +103,20 @@ func GetTaskDefinition(k8sclient client.Client, log logr.Logger, ctx context.Con
 	}
 	return definition, nil
 }
+
+func GetEvaluationDefinition(k8sclient client.Client, log logr.Logger, ctx context.Context, definitionName string, namespace string) (*klcv1alpha3.KeptnEvaluationDefinition, error) {
+	definition := &klcv1alpha3.KeptnEvaluationDefinition{}
+	err := k8sclient.Get(ctx, types.NamespacedName{Name: definitionName, Namespace: namespace}, definition)
+	if err != nil {
+		log.Error(err, "Failed to get KeptnEvaluationDefinition from application namespace")
```

Comment:
since we mention this like a normal/expected feature, are we sure we want to log error here?

Reply:
Agree, and I would also suggest to add the name of the evaluation definition and the namespace as properties, using the `keysAndValues` param of the logger function

Reply:
will adapt :+1:

Reply:
just realized we had the same in the task def redundant check, should we fix it here or in an other pr?